src/rpc/mining.cpp

```diff
@@ -194,6 +194,11 @@ UniValue generatetoaddress(const JSONRPCRequest& request)
 
     return generateBlocks(coinbaseScript, nGenerate, nMaxTries, false);
 }
+#else
+UniValue generatetoaddress(const JSONRPCRequest& request)
+{
+    throw std::runtime_error("Error RPC miner isn't compiled");
```

Comment:
```suggestion
    throw JSONRPCError(RPC_METHOD_NOT_FOUND, "This call is not available because RPC miner isn't compiled");
```

Reply:
@UdjinM6 Should this be more specific? Say `generatetoaddress is not available because RPC miner isn't compiled`? Same with below.

Reply:
I was looking at https://github.com/dashpay/dash/blob/develop/src/rpc/server.cpp#L561 as an example but yeah, there is also https://github.com/dashpay/dash/blob/develop/src/rpc/misc.cpp#L1092-L1093. I think either way is fine.